src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -18,16 +18,6 @@ categories :
       description : A category for CH4 OFI netmod variables
 
 cvars:
-    - name        : MPIR_CVAR_CH4_OFI_CAPABILITY_SETS_DEBUG
-      category    : CH4_OFI
-      type        : int
-      default     : 0
-      class       : none
-      verbosity   : MPI_T_VERBOSITY_USER_BASIC
-      scope       : MPI_T_SCOPE_LOCAL
-      description : >-
-        Prints out the configuration of each capability selected via the capability sets interface.
```

Comment:
If you're going to combine all of the debug CVARs into one, you need to at least keep the old ones around as aliases for a while. We have users who use these for debugging from time to time.

Reply:
That's a good idea.

Reply:
done.